tests/monitors/kubernetes_cluster/kubernetes_cluster_test.py

```diff
@@ -96,3 +96,39 @@ def test_kubernetes_cluster_namespace_scope(k8s_cluster):
             assert ensure_always(
                 lambda: not has_datapoint(agent.fake_services, dimensions={"kubernetes_namespace": "bad"})
             ), "got pod metrics from unspecified namespace"
+
+
+@pytest.mark.kubernetes
+def test_jobs(k8s_cluster):
```

Comment:
See if you can add a CronJob test, maybe using a very short interval so the test doens't take too long to run.

Reply:
Added a test, shortest interval available was 1m

Reply:
but the job should kickoff as soon as it's created, and the test should not need 1 more than 1 job run so it shouldn't even take that full minute.